packages/core/admin/admin/src/translations/en.json

```
@@ -709,7 +723,7 @@
   "content-manager.containers.ListPage.items": "{number, plural, =0 {items} one {item} other {items}}",
   "content-manager.containers.ListPage.table-headers.publishedAt": "State",
   "content-manager.containers.ListPage.selectedEntriesModal.title": "Publish entries",
-  "content-manager.containers.ListPage.selectedEntriesModal.selectedCount": "<b>{count}</b> {count, plural, =0 {entries} one {entry} other {entries}} selected",
+  "content-manager.containers.ListPage.selectedEntriesModal.selectedCount": "<b>{readyToPublishCount}</b> {readyToPublishCount, plural, =0 {entries} one {entry} other {entries}} ready to publish. <b>{withErrorsCount}</b> {withErrorsCount, plural, =0 {entries} one {entry} other {entries}} awaiting for action.",
```

I know this is what is in the design, but I think it is either: "awaiting action" or "waiting for action". Personally I prefer "waiting for".

```suggestion
  "content-manager.containers.ListPage.selectedEntriesModal.selectedCount": "<b>{readyToPublishCount}</b> {readyToPublishCount, plural, =0 {entries} one {entry} other {entries}} ready to publish. <b>{withErrorsCount}</b> {withErrorsCount, plural, =0 {entries} one {entry} other {entries}} waiting for action.",
```